src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.cs

```diff
@@ -788,7 +788,7 @@ public bool IsPrefix(ReadOnlySpan<char> source, ReadOnlySpan<char> prefix, Compa
 
                 if (!GlobalizationMode.Invariant)
                 {
-                    return StartsWithCore(source, prefix, options);
+                    return StartsWithCore(source, prefix, options, null /* matchLengthPtr */);
```

Comment:
nit: imo a named parameter looks better than a comment

```suggestion
                    return StartsWithCore(source, prefix, options, matchLengthPtr: null);
```